db/schema.rb

```diff
@@ -40,11 +42,18 @@
   end
 
   create_table "users", force: :cascade do |t|
-    t.string "fist_name"
+    t.string "first_name"
     t.string "last_name"
     t.date "date_of_birth"
     t.date "joining_date"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "role", default: 0
```

Comment:
maybe add a null: false validation for this?

Reply:
okey i did thank you for your input